charts/apisix/README.md

```diff
@@ -62,6 +62,8 @@ The following tables lists the configurable parameters of the apisix chart and t
 |------------------------------------------|-----------------------------------------------------|---------------------------------------------------------|
 | `apisix.enabled`                         | Enable or disable Apache APISIX itself              | `true`                                                  |
 | `apisix.enableIPv6`                      | Enable nginx IPv6 resolver                          | `true`                                                  |
+| `apisix.enableCustomizedConfig`          | Enable full customized `config.yaml`                | `false`                                                  |
+| `apisix.customizedConfig`                | If `apisix.enableCustomizedConfig` is true, full customized `config.yaml`  | `{}`                                                 |
```

Comment:
Note we should also say that once this field is enabled, other settings will be ignored.